### DISCUSSION TITLE
安装合辑包视觉反馈/希望扩充屏蔽功能

### DISCUSSION THREAD

**Discussion by Silvertraces** [the1812/Bilibili-Evolved#2649] [功能建议 / Ideas]
## 2. 希望扩充屏蔽类相关功能，包括但不限于：

- 首页、视频页的 课程推荐/小中高/考研/外语/考证考级视频(tag-cheese)
- 官方推广漫画/动画/游戏/直播/活动/创作大赛etc./（tag-blackboard/manga/douga/game/live/activity/m.bilibili.com/...）
- 特定类别视频（lol、wzry、生活区、无意义混剪/剪辑、短视频、特定视频标题过滤(e.g. 鬼知道我看了xx，迷惑行为大赏)）

**Comment by Geeyun-JY3** [comment]
首页的过滤记得 v1 的 `简化首页`（清爽 / 极简首页）是可以做到的（注意这和 v2 的 `简化首页` 功能不是一个东西）。现在得等这功能重写。

视频页推荐的这些屏蔽已经算是新功能了。

全站按标题、标签过滤相关讨论关联：#2298
